tests/test-webapps/test-jetty-webapp/pom.xml

```diff
@@ -141,6 +141,7 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-servlets</artifactId>
       <version>${project.version}</version>
+      <scope>provided</scope>
```

Comment:
Why `provided`? `jetty-servlets` is supposed to be portable across containers (it's not, but I think that's the idea), so it should really be in `WEB-INF/lib`.

Reply:
I don't think they are portable.  More over, the jetty-servlets module exposes them on the webapp classpath.  Putting them in webapp just brings in all our io, http and util classes and exposes all the classloader problems that entails.
In fact, some of the classes (eg PushCacheFilter) only work if loaded from the container.    I don't think we are in the business of providing general utility servlets for any container.